components/asset.rst

```diff
@@ -0,0 +1,251 @@
+.. index::
+   single: Asset
+   single: Components; Asset
+
+The Asset Component
+===================
+
+   The Asset component manages URL generation and versioning of web assets such
+   as CSS stylsheets, JavaScript files and image files.
+
+Installation
+------------
+
+You can install the component in two different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/asset`` on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/Asset).
+
+Usage
+-----
+
+Asset Packages
```

Comment:
Hmm, I believe we're starting to quick with this more complex terminology. On IRC, we have lots of people that are often very confused with the term "asset" itself. Maybe we should first talk about that and show a very basic Asset component usage.

Btw, I would also propose to make the `Package` component default to the `EmptyVersionStrategy`. I don't think many people need versioning at their first encounter. Then the example can be very simple:

````php
$package = new Package();

echo $package->getUrl('/image.png'); // /image.php
````